.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
quick q -- does jurity work with 3.12?

Reply:
I need to run some tests before confirming this.

Reply:
while at it, jumping full onboard would be nice to avoid going one-by-one

Reply:
Jurity now supports python 3.12